sql_user_check/all_tables_columns.sql

```diff
@@ -0,0 +1,52 @@
+SELECT
```

Comment:
This query can return a lot of results and we are not exactly using the result for downstream processes. Can we not do something like `SELECT 1 FROM ....` instead? Ditto below as well.